src/modules/contest/standings/standings.ts

```diff
@@ -0,0 +1,18 @@
+import { CodeforcesClient } from '../../../CodeforcesClient';
+import { ICodeforcesClient } from '../../../interfaces/codeforces-client.interface';
+import { Standings } from '../../../interfaces/standings.interface';
+
+type ContestStandingsParams = {
+  contestId: string;
+  from?: string;
+  count?: number;
+  handles?: Array<string>;
```

Comment:
`handles` accepts a semicolon-separated list, not an array.

Proposed solution:

```suggestion
/**
   * Semicolon-separated list of handles. No more than 10000 handles are accepted.
   */
  handles?: string;
```

The `JSDoc` comment here will provide guidance on what to send exactly in this param.